### PR TITLE
Fixes issue #14, refixes issues #12 and #17

### PR DESCRIPTION
**Pull request by hajgato**
Fixes issue #14, refixes issues #12 and #17

### CODE REVIEW

File: files/qsub.pl [hunk 1]
```diff
@@ -583,6 +587,29 @@ sub parse_script
         }
     };
 
+    # handle queues
+    if ($orig_argsh{'q'}) {
```
Comment (stdweird) [inline files/qsub.pl]: i would warn the same message for both arg and/or directive

File: files/qsub.pl [hunk 2]
```diff
@@ -583,6 +587,29 @@ sub parse_script
...
+        warn "Please do not use -q <QUEUE> option for qsub.\n" .
+            "Right now it sets only the default walltime based on the queue,\n" .
+            "So please request explicitly the desired walltime.";
+
+    }
+    my $userqueue = $destination || $set{'q'};
+    if ($userqueue) {
+        my $modwalltime;
+        switch ($userqueue) {
+            case "short"   { $modwalltime = '1:00:00'; }
```
Comment (stdweird) [inline files/qsub.pl]: use a hash to map this

File: files/qsub.pl [hunk 3]
```diff
@@ -583,6 +587,29 @@ sub parse_script
...
+            case "bshort"  { $modwalltime = '1:00:00'; }
+            case "long"    { $modwalltime = '72:00:00'; }
+            case "debug"   { $modwalltime = '15:00'; }
+            else           { fatal("You have used a non-existing queue name!\n"); }
+        }
+        splice(@newtxt, 1, 0, "#PBS -l walltime=$modwalltime");
```
Comment (stdweird) [inline files/qsub.pl]: insert a comment like `## walltime from deprecated queue $userqueue`

File: files/qsub.pl (hunk 1 above)
Comment (hajgato) [reply files/qsub.pl]: Done.

File: files/qsub.pl (hunk 3 above)
Comment (hajgato) [reply files/qsub.pl]: Done.

File: files/qsub.pl (hunk 2 above)
Comment (hajgato) [reply files/qsub.pl]: Done.

File: files/qsub.pl [hunk 4]
```diff
@@ -583,6 +583,29 @@ sub parse_script
         }
     };
 
+    # handle queues
+    if ($orig_argsh{'q'} || $set{'q'}) {
+        warn "Please do not set a queue (\'-q\' otion or \'#PBS -q\' directive).\n" .
```
Comment (stdweird) [inline files/qsub.pl]: typo `otion`

File: files/qsub.pl [hunk 5]
```diff
@@ -583,6 +583,29 @@ sub parse_script
...
+    if ($userqueue) {
+        my %modwalltime = (
+            'short'  => '1:00:00',
+            'bshort' => '1:00:00', 
+            'long'   => '72:00:00',
+            'debug'  => '15:00',
+        );
+        if ($modwalltime{$userqueue}) {
+            splice(@newtxt, 1, 0, "## walltime from deprecated queue $userqueue", "#PBS -l walltime=$modwalltime{$userqueue}");
+            } else { fatal("You have used a non-existing queue name!\n"); }
```
Comment (stdweird) [inline files/qsub.pl]: fix indent, use multiple lines/indentation for the else block

**Comment by stdweird** [comment]
@hajgato some minro remarks, looks good otherwise

File: files/qsub.pl (hunk 4 above)
Comment (hajgato) [reply files/qsub.pl]: fixed

**Comment by stdweird** [comment]
@hajgato thanks, don't forget to make rpm

**Pull request merged**